ui/DesignSystem/Primitive/Input/Password.svelte

```diff
@@ -0,0 +1,127 @@
+<script>
```

Comment:
Note: adding this component only because Svelte didn't allow me to make the input type variable:
```[svelte invalid-type] [E] 'type' attribute cannot be dynamic if input uses two-way binding```.